### DISCUSSION TITLE
Template API and Categories

### DISCUSSION THREAD

**Discussion by castnettech** [Sellbrite/Sellbrite-API#42] [Q&A]
I have built a robust integration from a spreadsheet that syncs with sellbrite for a client, however sellbrite doesn’t seem to support channel categories, they’re self defined through recipes which cannot obviously be created for every category in a sales channel (i.e. eBay) . I have pulled the eBay categories directly in the spreadsheet but there’s no way to transmit that to sellbrite.

the main reason I need this is to auto apply templates, this creates a unnecessary manual step for users of the api.

What’s the best way to deal with post products in different categories and have the same template applied to them all regardless of category??

**Comment by castnettech** [comment]
Is there a better way to apply templates, this would be a great feature for api users?

**Comment by rengler-godaddy** [comment]
Hello, 

Templates can’t be applied to a listing using the API. Templates along with creating listings are managed within the Sellbrite dashboard. The following help article can go over how to manage templates and recipes: https://support.sellbrite.com/en/articles/3367215-get-started-with-templates-recipes

If you have further questions about managing listings and templates feel free to contact our support team at support@sellbrite.com
